doc/source/config.rst

```diff
@@ -115,6 +115,26 @@ It can either be ``enabled`` or ``disabled``.
   sp.auto_cookie_secure.enable();
   sp.auto_cookie_secure.disable();
 
+cookie_samesite
+^^^^^^^^^^^^^^^^
+ * `default: disabled`
+ 
+``samesite`` will add the `samesite <https://tools.ietf.org/html/draft-west-first-party-cookies-07>`_
+attribute to cookies. It `prevents CSRF <https://chloe.re/2016/04/13/goodbye-csrf-samesite-to-the-rescue/>`_
```

Comment:
why not use "https://www.owasp.org/index.php/SameSite" as a link instead of a blogpost that might disappear ?

Reply:
Because I'm not a big fan of the OWASP :>
But sure, I'll do it :)